tests/field/example-django.sh

```diff
@@ -6,6 +6,14 @@
 # The periodic regeneration is configured as a scheduled
 # pipeline in GitLab > CI/CD > Schedules.
 #
+# To run this field test locally:
+#  (1) Generate a Personal Access Token on GitLab 
+#      Top-right usesr menu > Settings > Access Tokens
+#  (2) export GITLAB_API_TOKEN=<your personal access token>
+#  (3) run ./tests/field/example-django.sh
```

Comment:
"run this script" is probably sufficient. Then we can simply copy+paste this for future scripts w/o the need to update the file name it refers to. What do you think?